### ISSUE TITLE
Building is broken for Svelte

### ISSUE THREAD

**Issue by nickisnoble** [pikapkg/create-snowpack-app#162]
`npm run build` does not fail, but the resulting build references a stylesheet `App.css.js` that doesn't exist – so the page doesn't actually show when served.

Attempting to use the webpack plugin fails for a similar reason:

```
▼ Console

  [log] ! optimizing dependencies...
  [log]
    ⦿ web_modules/           size       gzip       brotli
      └─ svelte/internal.js  51.67 KB   13.02 KB   11.26 KB


  [error]
  [error] ./_dist_/App.js
  Module not found: Error: Can't resolve './App.css.js' in '/Users/nickisnoble/Code/new-dir/.build/_dist_'
  resolve './App.css.js' in '/Users/nickisnoble/Code/new-dir/.build/_dist_'
    using description file: /Users/nickisnoble/Code/new-dir/package.json (relative path: ./.build/_dist_)
      Field 'browser' doesn't contain a valid alias configuration
      using description file: /Users/nickisnoble/Code/new-dir/package.json (relative path: ./.build/_dist_/App.css.js)
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js doesn't exist
        .wasm
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.wasm doesn't exist
        .mjs
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.mjs doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.js doesn't exist
        .json
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.json doesn't exist
        as directory
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js doesn't exist
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.wasm]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.mjs]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.js]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.json]
   @ ./_dist_/App.js 1:0-22
   @ ./_dist_/index.js
  [log]  5 assets

▶ Build Complete!
```

The above is all from a fresh run of CSA, with no other modification past these two steps (let me know if I missed something):

1. `npm install @snowpack/plugin-webpack -D`
2. Update snowpack config:
```
{
  "extends": "@snowpack/app-scripts-svelte",
  "scripts": {},
  "plugins": ["@snowpack/plugin-webpack"]
}
```

**Comment by FredKSchott** [comment]
We just fixed an issue related to this this morning, can you update Snowpack to latest and retry?

**Comment by nickisnoble** [comment]
@FredKSchott thanks for the fast reply! Still seems to be happening after update to `2.6.2`

**Comment by FredKSchott** [comment]
Okay, thanks for filing. Tracking the fix for this issue here: https://github.com/pikapkg/snowpack/pull/561

**Comment by FredKSchott** [comment]
Released in v2.6.3! https://github.com/pikapkg/snowpack/releases/tag/v2.6.3 Can confirm that the svelte CSA app is now back to working in the unbundled build. Thanks again for filing.

**Comment by nickisnoble** [comment]
🎉 Thanks!!
